src/lib/utils/multicall.ts

```diff
@@ -329,17 +332,31 @@ export class MultiCaller {
         input.push({
           targetAddr: t,
           encoder: () => erc20Iface.encodeFunctionData('name'),
-          decoder: (returnData: string) =>
-            erc20Iface.decodeFunctionResult('name', returnData)[0] as string,
+          decoder: (returnData: string) => {
+            if (isBytes32(returnData)) {
```

Comment:
Lets add a comment here about why we do this (maker doesnt follow the erc20 spec), and include a link to the maker contracts